ui/narrative/methods/annotate_contigs/spec.json

```diff
@@ -23,11 +23,29 @@
         },
         {
             "id": "scientific_name",
-            "optional": true,
+            "optional": false,
             "advanced": false,
             "allow_multiple": false,
-            "default_values": [ "" ],
-            "field_type" : "text"
+            "default_values": [""],
+            "field_type": "dynamic_dropdown",
+            "dynamic_dropdown_options": {
+                "data_source": "custom",
+                "service_function": "taxonomy_re_api.search_taxa",
+                "service_version": "dev",
+                "service_params": [
+                    {
+                        "search_text": "prefix:{{dynamic_dropdown_input}}",
+                        "ns": "ncbi_taxonomy",
+                        "limit": 1000
```

Comment:
I would limit this to 100 or 50 or 20 instead. You may get a performance benefit, and no way are they going to scroll through 1000

Reply:
sounds good